bot/convert_currency.go

```diff
@@ -63,12 +77,13 @@ func ConvertMessage(message string) string {
 
 	// check amount was provided
 	if arraySize > 3 {
```

Comment:
On some countries comma are the rule to split cents.... 
The ideal world here would be create an validation based in some system variable. 

1. Detect the user's language or locale.
2. Create a currency validation function considering comma or dot as the separator for cents.